index.js

```diff
@@ -30,25 +30,32 @@ function fastifySwagger (fastify, opts, next) {
         },
         response: {
           200: {
-            description: 'The swagger definition of the routes',
-            type: 'object',
-            properties: {
-              hello: { type: 'string' }
-            }
+            description: 'The swagger definition of the routes'
```

Comment:
We can't, the response **must** have the description property, if we don't set it by hand, this plugin will set it at `"Default response"`.